packages/api-v4/src/kubernetes/types.ts

```diff
@@ -57,7 +57,7 @@ export interface KubernetesDashboardResponse {
 }
 
 export interface ControlPlaneOptions {
-  high_availability: boolean;
+  high_availability: boolean | undefined;
```

Comment:
Now that I think about it, this notation might be more appropriate, although they accomplish the same thing. 

```suggestion
  high_availability?: boolean;
```

Reply:
Yeah, this makes sense.